CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [1.0.1] - 2018-04-16
```

Comment:
I will go ahead and merge this but in general per our [guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#unreleased-change) you submit the changelog without the date or version there are several reasons for this:
- PRs are not a FIFO queue, there is no guarantee on the ordering of PRs merged
- we date the release of the new version rather than when it was submitted as there is no SLA on review times
- maintainer may disagree with you on the versioning, in this case as there was no functional change I would not generally cut a release and wait for the next functional change to be submitted.